### PR TITLE
GameDB: Cleanup names ( + add serials)

### PR DESCRIPTION
**Pull request by RedDevilus**
I was expecting to add some more serials to SLUS (NTSC) but it seems to be more well maintained than the SLES (PAL). Most additions are demos but there are some normal full entries . Minor fixups on SLUS and other regions for names.

### CODE REVIEW

**Comment by RedDevilus** [comment]
One thing which Sony lacks are consistent official names, even websites such as redump or others like it have a different name for it, even Wikipedia has several variant on game names. Looking at the side is already different from front cover and also what is stated on the disc. I'm not even begun talking about the other region discs for the same game, hell even for the same region but with a different serial it can already differ.

Tl;dr Lots of game names are wrong no matter what if you want official names.

Sources:
https://psxdatacenter.com/psx2/sitenews2.html
https://www.serialstation.com/serials/?system=2&regions_ch=false&regions_op=OR
http://www.thecoverproject.net/view.php?cat_id=6
....

File: bin/GameIndex.yaml [hunk 1]
```diff
@@ -39516,6 +39599,21 @@ SLUS-29197:
 SLUS-29198:
   name: "Guitar Hero II [Demo]"
   region: "NTSC-U"
+SLUS-29199:
+  name: "dot Hack G.U. Vol. 1 - Rebirth [Demo]"
```
Comment (tadanokojin) [inline bin/GameIndex.yaml:39603]: **.hack//G.U. Vol. 1//Rebirth**
[Wikipedia](https://en.wikipedia.org/wiki/.hack//G.U.#Vol._1//Rebirth)
[IGN](https://www.ign.com/articles/2007/10/05/hackgu-vol1rebirth-review)
[GameStop](https://www.gamestop.com/video-games/retro-gaming/playstation-2/games/products/.hack--g.u.-vol.-1--rebirth/10168364.html?rrec=true)
[Metacritic](https://www.metacritic.com/game/playstation-2/hackgu-vol-1rebirth)

**Comment by lightningterror** [comment]
Status?

**Comment by RedDevilus** [comment]
I'll undraft when i have time to do the big cleanup but i haven't had the time to finish it.

**Comment by RedDevilus** [comment]
Should be ready now :)

File: bin/GameIndex.yaml [hunk 2]
```diff
@@ -33363,15 +33363,15 @@ SLUS-20586:
...
+  name: "Driv3r"
   region: "NTSC-U"
   compat: 5
 SLUS-20588:
   name: "Activision Anthology"
   region: "NTSC-U"
   compat: 5
 SLUS-20590:
-  name: "Spy Hunter 2"
+  name: "SpyHunter 2"
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:33374]: It's two words https://lh3.googleusercontent.com/proxy/E1PTf4Tb3ZZ5QL73RbvgtsTco3k21aKpqgQLGqSlKLcl9TDAf1spNMzHC6ZixQsjQ8_uV-rAb50WvuZWK47LMCHbOkTujoh4yQxVGrmwGa1BcDrycwA5rI621ekVGTAkoA3DCVDxI5qpi8aaVJxENT5kEdu9cA

File: bin/GameIndex.yaml [hunk 3]
```diff
@@ -33582,7 +33582,7 @@ SLUS-20640:
   region: "NTSC-U"
   compat: 5
 SLUS-20641:
-  name: "IndyCar Series"
+  name: "IndyCar Series featuring the Indianapolis 500"
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:33585]: As far as I can tell, it's just IndyCar Series

https://steemitimages.com/640x0/https://cdn.steemitimages.com/DQmQxhbr2QpeNy1FTAqXtKYzu6zRiQy474KzjqZuafRM77u/image.png

File: bin/GameIndex.yaml [hunk 4]
```diff
@@ -33827,7 +33827,7 @@ SLUS-20702:
   region: "NTSC-U"
   compat: 5
 SLUS-20703:
-  name: "Air Force - Delta Strike"
+  name: "AirForce: Delta Strike"
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:33830]: bit difficult because the cover is in all caps, but generally Airforce is one word, no capitalisation in the middle

Example: https://www.covercentury.com/covers/ps2/e/energy_airforce_-_aim_strike_d.jpg

File: bin/GameIndex.yaml [hunk 5]
```diff
@@ -34078,7 +34078,7 @@ SLUS-20761:
   region: "NTSC-U"
   compat: 5
 SLUS-20762:
-  name: "Secret Weapons Over Normandy"
+  name: "Secret Weapons over Normandy"
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:34081]: Over is capitalised as earlier

File: bin/GameIndex.yaml [hunk 6]
```diff
@@ -34227,7 +34227,7 @@ SLUS-20798:
   name: "Starcraft - Ghost"
   region: "NTSC-U"
 SLUS-20799:
-  name: "Terminator, The - Rise of the Machines"
+  name: "Terminator 3, The: Rise of the Machines"
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:34230]: Not your mistake but there is no "The" on Terminator 3
https://i.ytimg.com/vi/Llp2QYdj8gk/maxresdefault.jpg

File: bin/GameIndex.yaml [hunk 7]
```diff
@@ -34303,7 +34303,7 @@ SLUS-20821:
   region: "NTSC-U"
   compat: 5
 SLUS-20822:
-  name: "Galactic Wrestling"
+  name: "Galactic Wrestling: The Kinnikuman Legacy"
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:34306]: The actual name is "Galactic Wrestling: Featuring Ultimate Muscle - The Kinnikuman Legacy"
Your choice on this one ;)

https://i109.fastpic.ru/big/2019/0320/33/_59492eafcc2ecfce6cdcf975ab804833.jpg?noht=1&tk=d301&md5=ZgUe7otgsKweMLQD2wt9og&expires=1613930400

File: bin/GameIndex.yaml [hunk 8]
```diff
@@ -35696,7 +35696,7 @@ SLUS-21148:
   region: "NTSC-U"
   compat: 5
 SLUS-21149:
-  name: "Yourself Fitness"
+  name: "Yourself!Fitness"
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:35699]: missing space after the !

https://capasecovers.files.wordpress.com/2010/01/yourselffitness-ps2.jpg

File: bin/GameIndex.yaml [hunk 9]
```diff
@@ -36161,7 +36161,7 @@ SLUS-21248:
   clampModes:
     vuClampMode: 3  # Fixes broken polygons on trees.
 SLUS-21249:
-  name: "Yourself Fitness - Lifestyle"
+  name: "Yourself!Fitness - Lifestyle"
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:36164]: Space after the ! as before.

File: bin/GameIndex.yaml [hunk 10]
```diff
@@ -36787,7 +36787,7 @@ SLUS-21381:
   clampModes:
     vuClampMode: 2
 SLUS-21382:
-  name: "Franklin - A Birthday Surprise"
+  name: "Franklin the Turtle - A Birthday Surprise"
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:36790]: Looks more like "Franklin-Turtle: Franklin - A Birthday Surprise" to me, or just how it was

https://my-test-11.slatic.net/p/fffb15b474ff4bed5549806587dba0c6.jpg

File: bin/GameIndex.yaml [hunk 11]
```diff
@@ -36872,11 +36872,11 @@ SLUS-21401:
   region: "NTSC-U"
   compat: 4
 SLUS-21402:
-  name: "Micro Machines v4"
+  name: "Micro Machines V4"
   region: "NTSC-U"
   compat: 5
 SLUS-21403:
-  name: "Backyard Baseball '07"
+  name: "Backyard Baseball 2007"
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:36879]: Actually "Backyard Sports - Baseball 2007"

https://lh3.googleusercontent.com/proxy/by8ceHDADzvqcMH9tezxwg5J5NB1GWmaEYTmG2E0h4Noqgfmg-rl3XuWSc_zb-aDzqQImotAm70yC0LfnD6C_7SPiGxYowgsuXUTLxyUIOe9poMhuDywt4T_E-J7HtSG8x6o8cC0xc_GUh2iTJeq5D4nCvAdm3Zr_-MSEvOL

File: bin/GameIndex.yaml [hunk 12]
```diff
@@ -39441,17 +39519,20 @@ SLUS-29170:
...
-  name: "Namco Transmission Demo Disc Vol.3.1"
+  name: "Namco Transmission Demo Disc Vol. 3.1"
   region: "NTSC-U"
 SLUS-29175:
-  name: "Namco Transmission Demo Disc Vol.3.2"
+  name: "Namco Transmission Demo Disc Vol. 3.2"
   region: "NTSC-U"
 SLUS-29178:
-  name: "TOCA Race Driver 3 [Demo]"
+  name: "TOCA Race Driv3r [Demo]"
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:39535]: As before Driv3r only exists in the Driver universe, not every game with "Driver 3" in the title.

**Review by refractionpcsx2 — CHANGES_REQUESTED** [review]

File: bin/GameIndex.yaml (hunk 10 above)
Comment (RedDevilus) [reply bin/GameIndex.yaml:36790]: image

Enhance :)

Comment (refractionpcsx2) [reply bin/GameIndex.yaml:36790]: huh, it does say "the", but still, it has "Franklin" again as part of the main title, so it's still wrong ;)

**Comment by RedDevilus** [comment]
image
+ 32 new entries
